active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,220 @@
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+Number box is a numerics only text control with support for input formatting, validation, basic calculation, and increment stepping.
+
+**Important APIs:** [NumberBox class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.numberbox)
+
+## Is this the right control? 
+
+Use a **NumberBox** control to recieve and display formatted numeric values such as currency, IP addresses, and telephone numbers. A **NumberBox** can also be enabled with basic calculator support to compute multiplication, division, addition, and subtraction across order of operations on entered equations. 
+
+## Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+### Create a NumberBox
+
+XAML
+```XAML
+<NumberBox AutomationProperties.Name="simple NumberBox"/>
+```
+
+### Add placeholder text and a header
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="simple NumberBox" Header="Enter equation:" PlaceholderText="A + B * C" />
+```
+
+### Enable calculation support
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for simple calculations" 
+    Header="Enter equation:" 
+    PlaceholderText="A + B * C" 
+    AcceptsCalculation="True" />
+```
+
+### Add formatting
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for IP addresss" 
+    Header="Enter equation:" 
+    PlaceholderText="000.000.000.000"
+    FormatType="IPAddress" />
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for international telephone" 
+    Header="Enter equation:" 
+    PlaceholderText="0 (123) 456-7890"
+    FormatType="InternationalTelephone" />
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for custom input" 
+    Header="Enter equation:" 
+    PlaceholderText="00 ft 00 in"
+    CustomFormatString="## ft ## in"
+    StripLeadingZeros="False" />
```

Comment:
This example implies that the default value it true, is that true?

Reply:
That's what I initially intended. Do you think NumberBox should not automatically trim leading zeros? The push on the proposal right now is for it to support purely mathematical intentions rather than numerical strings such as phone numbers or zip codes.

Reply:
I see, makes sense.